runtime

```diff
@@ -8,18 +8,21 @@
 #     runtime run                        (to run run.sh)
 #     runtime test integration/tc_68_2   (to run the automated test tc_68_2)
 
-set -e  # Makes any errors exit the script
-
 RUNTIME_DIR=$(dirname "${BASH_SOURCE[0]}")
 
 cd $RUNTIME_DIR
 
 case $1 in
     build)
```

Comment:
I remember making a comment about this, but it seems to have gotten resolved.. what was the rationale for having arguments to `runtime build`?

Reply:
I answered on the previous comment about it. one of the beginning ones